### ISSUE TITLE
[Bug]: Deleting a bottle does not remove its library entries, causing Bottles to not launch

### ISSUE THREAD

**Issue by AshtonFerns** [bottlesdevs/Bottles#2386]
### Describe the bug

When a bottle is deleted, the library entries for its installed apps are not removed from Bottles. Bottles will not start after relaunching, with the following output:

```python
23:42:07 (INFO) Library saved 
Traceback (most recent call last):
  File "/app/share/bottles/bottles/frontend/windows/main_window.py", line 155, in set_manager
    self.page_library = LibraryView(self)
  File "/app/share/bottles/bottles/frontend/views/library.py", line 43, in __init__
    self.update()
  File "/app/share/bottles/bottles/frontend/views/library.py", line 56, in update
    entry = LibraryEntry(self, u, e)
  File "/app/share/bottles/bottles/frontend/widgets/library.py", line 62, in __init__
    self.__remove_entry()
  File "/app/share/bottles/bottles/frontend/widgets/library.py", line 137, in __remove_entry
    self.library.remove_entry(self)
  File "/app/share/bottles/bottles/frontend/views/library.py", line 66, in remove_entry
    entry.hide()
  File "/app/share/bottles/bottles/frontend/widgets/library.py", line 169, in hide
    self.get_parent().set_visible(False)
AttributeError: 'NoneType' object has no attribute 'set_visible'
```

To fix the issue and start Bottles again, the problematic library entry in ~/.var/app/com.usebottles.bottles/data/bottles/library.yml must be removed. 

### To Reproduce

1. Create a new bottle
2. Add a shortcut to an app
3. Add the app to library
4. Delete the bottle
5. The library entry is not removed, and relaunching Bottles will cause it to not start. 

### Package

Flatpak from Flathub

### Distribution

Arch Linux

### Troubleshooting Logs

```shell
Official Package: true
Version: 2022.12.14
Display:
    X.org: true
    X.org (port): :0
    Wayland: false
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 6.0.12-arch1-1
Distro:
    Name: GNOME
    Version: '43'
Disk:
    Total: 8314679296
    Free: 8314531840
RAM:
    MemTotal: 15.5GiB
    MemAvailable: 9.9GiB
Bottles_envs: null
```

### Display Protocol

X11

### Display Backend used by Bottles

X11

**Comment by mirkobrombin** [comment]
Fixed in 2022.12.14.1

**Comment by ruzko** [comment]
Either it wasn't completely fixed in 2022.12.14.1, or I'm experiencing a different bug.
Bottles does start, but the library entry isn't removed. It's just replaced by a grey slate:

![Screenshot_20221229_221504](https://user-images.githubusercontent.com/75946143/210012119-e2f0e28c-c219-415a-9327-581c55c2ff89.jpeg)


### Debugging information
```
Official Package: true
Version: 2022.12.14.1
Display:
    X.org: true
    X.org (port): :1
    Wayland: true
Graphics:
    vendors:
        nvidia: &amp;id002
            vendor: nvidia
            envs:
                __NV_PRIME_RENDER_OFFLOAD: '1'
                __GLX_VENDOR_LIBRARY_NAME: nvidia
                __VK_LAYER_NV_optimus: NVIDIA_only
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/nvidia_icd.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/nvidia_icd.json
        intel: &amp;id001
            vendor: intel
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/intel_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/intel_icd.i686.json
    prime:
        integrated: *id001
        discrete: *id002
Kernel:
    Type: Linux
    Version: 6.0.14-300.fc37.x86_64
Distro:
    Name: GNOME
    Version: '43'
Disk:
    Total: 16681820160
    Free: 16681648128
RAM:
    MemTotal: 31.1GiB
    MemAvailable: 5.3GiB
Bottles_envs: null
```

### Log of flatpak command
```
flatpak run com.usebottles.bottles

(process:2): Gtk-WARNING **: 22:21:01.084: Locale not supported by C library.
	Using the fallback 'C' locale.
22:21:01 (INFO) Bottles Started!
22:21:04 (INFO) Catalog components loaded
22:21:04 (INFO) Catalog installers loaded
22:21:04 (INFO) Catalog dependencies loaded
22:21:04 (INFO) Performing Bottles checks…
22:21:04 (INFO) Steam path doesn't exist, creating now.
22:21:04 (INFO) Dxvks found:
	 - dxvk-2.0
	 - x32
	 - x64
	 - dxvk-async-2.0
	 - dxvk-2.0-1-12901b5

22:21:04 (INFO) Vkd3ds found:
	 - setup_vkd3d_proton.sh
	 - x64
	 - x86
	 - vkd3d-proton-2.8

22:21:04 (INFO) Nvapis found:
	 - dxvk-nvapi-v0.5.4
	 - dxvk-nvapi-v0.6

22:21:04 (INFO) Latencyflexs found:
	 - latencyflex-v0.1.1

22:21:04 (INFO) Runners found:
	 - sys-wine-7.0.1
	 - caffe-7.20
	 - wine-ge-proton7-35

22:21:05 (INFO) Temp directory cleaned successfully!
22:21:05 (INFO) Bottles found:
	 - games3


(bottles:2): Gtk-CRITICAL **: 22:21:05.484: gtk_widget_set_parent: assertion '_gtk_widget_get_parent (widget) == NULL' failed
22:21:05 (INFO) Library saved
```
